components/alert/alert-toast.js

```diff
@@ -144,7 +144,7 @@ class AlertToast extends LitElement {
 	render() {
 		return html`
 			<div
-				class="d2l-alert-toast-container"
+				class="d2l-alert-toast-container vdiff-target"
```

Comment:
Do we want to leave this here in case anyone ever wraps the toast and needs a way to target just that component? Or remove for now and add back if that happens?

Reply:
I'm good either way!